google/generativeai/discuss.py

```diff
@@ -504,6 +504,11 @@ def count_message_tokens(
     model: str = DEFAULT_DISCUSS_MODEL,
     client: Optional[glm.DiscussServiceAsyncClient] = None,
 ):
+    """
```

Comment:
You want a 1-line description on the first line with the """.

Please also include the `Args` and `returns` block.

Reply:
Should the args block include the same descriptions included while describing the `ChatResponse` object? I can change them if needed.

Reply:
You probably don't need the full descriptions, you can point back to `chat`, but we I would copy the text from prompt, since this point is important: 

```
prompt: You may pass a `types.MessagePromptOptions` **instead** of a
      setting `context`/`examples`/`messages`, but not both.
```